src/util/os_utils.js

```diff
@@ -975,6 +1023,7 @@ exports.read_drives = read_drives;
 exports.get_raw_storage = get_raw_storage;
 exports.remove_linux_readonly_drives = remove_linux_readonly_drives;
 exports.get_main_drive_name = get_main_drive_name;
+exports.df = _df;
```

Comment:
this export is only meant for tests, I wonder if we should export it as an internal name
```suggestion
exports._df = _df;
```
I am not that opinionated here, but if we want to pass the intention that this function is only meant for this file's internal use, then the export should provide the same intention.

I feel it is similar to writing an internal function inside a class and exporting the class.
@guymguym WDYT?

Reply:
I have changed it to be _df. @guymguym any comments?